src/constants/theme/fonts.ts

```diff
@@ -1,4 +1,14 @@
+import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
+
 export const FONTS = {
   contentFontBold: 'SourceSansPro-SemiBold',
-  contentFontRegular: 'SourceSansPro-Regular'
+  contentFontRegular: 'SourceSansPro-Regular',
+  defaultFontWeight: 600,
+  lightFontWeight: 400,
+  capsLetterSpacing: '0.4px',
+  listTitleLetterSpacing: '0.11px',
+  smallFontSize: `${wp('3%')}px`,
+  defaultFontSize: `${wp('4%')}px`,
+  largeFontSize: `${wp('4.5%')}px`,
+  headingFontSize: `${wp('5%')}px`
```

Comment:
couldn't really come up with  a better naming here. I am open for any ideas

Reply:
i think heading is fine, but if you are unhappy anything else i have in mind is extraLarge